index.bs

```diff
@@ -1553,6 +1575,41 @@ Issue: Determine proper charset-handling for the JSON header value.
 
 <h3 id="processing-an-attribution-source">Processing an attribution source</h3>
 
+To <dfn>check if an [=attribution source=] exceeds the time-based destination limits</dfn> given an
+[=attribution source=] |source|, run the following steps:
+
+1. Let |matchingSources| be all [=attribution rate-limit records=] |record| in the [=attribution rate-limit cache=] where all of the following are true:
+     * |record|'s [=attribution rate-limit record/scope=] is "<code>[=rate-limit scope/source=]</code>"
+     * |record|'s [=attribution rate-limit record/source site=] and |source|'s [=attribution source/source site=] are equal
+     * |record|'s [=attribution rate-limit record/expiry time=] is greater than |source|'s [=attribution source/source time=]
+     * The [=duration from=] |record|'s [=attribution rate-limit record/time=] and |source|'s [=attribution source/source time=]
+        is less than [=destination rate-limit window=] 
+1. Let |matchingSameReportingSources| be all the [=attribution rate-limit records|records=] in |matchingSources|
+    whose associated [=attribution rate-limit record/reporting origin=] is [=same site=] with |source|'s 
+    [=attribution source/reporting origin=].
+1. Let |destinations| be the [=set=] of every [=attribution rate-limit record/attribution destination=] in |matchingSources|,
+    [=set/union|unioned=] with |source|'s [=attribution source/attribution destinations=].
+1. Let |sameReportingDestinations| be the [=set=] of every [=attribution rate-limit record/attribution destination=] in |matchingSameReportingSources|,
+    [=set/union|unioned=] with |source|'s [=attribution source/attribution destinations=].
+1. Let |hitRateLimit| be whether |destinations|'s [=set/size=] is greater than [=max destinations per rate-limit window=][0].
+1. Let |hitSameReportingRateLimit| be whether |sameReportingDestinations|'s [=set/size=] is greater than [=max destinations per rate-limit window=].
+1. If (|hitRateLimit|, |hitSameReportingRateLimit|) is
+    <dl class="switch">
+    : (false, false)
+    :: Return "<code>[=destination rate-limit result/allowed=]</code>".
+    : (false, true)
+    :: Return "<code>[=destination rate-limit result/hit reporting limit=]</code>".
+    : (true, false)
+    :: Return "<code>[=destination rate-limit result/hit global limit=]</code>".
+    : (true, true)
+    :: Return "<code>[=destination rate-limit result/hit reporting limit=]</code>".
+
+    </dl>
+
+Note: We do not emit an [=attribution debug report=] for "<code>[=destination rate-limit result/hit global limit=]</code>",
```

Comment:
We do emit an attribution debug report, but just not the explicit report type. Shall we make this a bit more clear?

Reply:
Done